py-polars/src/series.rs

```diff
@@ -41,27 +41,39 @@ impl PySeries {
     }
 }
 
-pub(crate) fn to_series_collection(ps: Vec<PySeries>) -> Vec<Series> {
-    // prevent destruction of ps
-    let mut ps = std::mem::ManuallyDrop::new(ps);
+pub(crate) trait ToSeries {
+    fn to_series(self) -> Vec<Series>;
+}
+
+impl ToSeries for Vec<PySeries> {
+    fn to_series(self) -> Vec<Series> {
```

Comment:
Can we do the `mem::transmute` with the safety statement that we have a transparent repr.

This works in both directions and is much clearer in this case.